namui/namui-cli/src/services/rust_build_service.rs

```diff
@@ -133,6 +133,10 @@ fn get_envs(build_option: &BuildOption) -> Vec<(&str, &str)> {
         envs.push(("NAMUI_CFG_WATCH_RELOAD", ""));
     }
 
+    if !build_option.release {
+        envs.push(("RUST_BACKTRACE", "1"));
+    }
+
```

Comment:
This flag seems for debug, but I think it's okay to stay. What do you think?

Reply:
It's intentional. only for non-release mode